ctru-rs/src/smdh.rs

```diff
@@ -0,0 +1,99 @@
+use crate::services::cfgu::Language;
+use bitflags::bitflags;
+
+#[derive(Copy, Clone, Debug)]
+#[repr(C)]
+pub struct Smdh {
```

Comment:
A doc comment here linking to 3dbrew.org might be useful.

Also, this is a huge struct so deriving `Copy` is probably not ideal. I think we'd want people to opt-in to explicit `clone()` rather than automatically copying

Reply:
+1 on removing Copy